samples/java-customer-registry/src/main/java/customer/CustomerValueEntity.java

```diff
@@ -28,6 +29,12 @@
  */
 public class CustomerValueEntity extends CustomerValueEntityInterface {
 
+  private final String entityId;
+
+  public CustomerValueEntity(ValueEntityContext context) {
+    this.entityId = context.entityId();
+  }
```

Comment:
users should be able to define the constructor the way they prefer. By default, we generated one getting the `ValueEntityContext` and setting the `entityId`.